yax/play/src/main/scala/org/lyranthe/prometheus/client/integration/play/filters/PrometheusFilter.scala

```diff
@@ -22,7 +22,7 @@ class PrometheusFilter @Inject()(implicit
 ) extends Filter {
   private final val ServerErrorClass = "5xx"
 
-  private final val RouteRegex = "^[/a-zA-Z0-9$_\\-]+$".r
+  private final val RouteRegex = "^/([^<]*)".r
```

Comment:
How about including `/` inside the parentheses? So that it is part of the matched path.